src/main/java/com/checkmarx/flow/service/RepoService.java

```diff
@@ -37,17 +37,16 @@ public void sendMergeComment(ScanRequest request, String comment){
                     return;
                 }
                 log.debug("sendMergeComment: Going to update {} pull request comment",
-                          request.getRepoType());
+                        request.getRepoType());
                 updateComment(commentToUpdate.getCommentUrl(), comment, request);
             } else {
```

Comment:
What are the changes in this file? not clear

Reply:
optimization from IDEA